circle.yml

```diff
@@ -26,7 +28,7 @@ dependencies:
       export PATH="$HOME/miniconda/bin:$PATH" &&
       source activate test &&
       conda install ciocheck -c spyder-ide &&
-      conda install spyder &&
+      pip install git+https://github.com/spyder-ide/spyder@3.x &&
```

Comment:
ah, I did have a look and I was wondering why it was so complicated :) Thanks for the explanation.